README.md

```diff
@@ -1,191 +1,132 @@
-# Molecule Hetzner Cloud Plugin
+# Molecule Hetzner Cloud Driver
 
-[![PyPI Package](https://badge.fury.io/py/molecule-hetznercloud.svg)](https://badge.fury.io/py/molecule-hetznercloud)
-[![Repository License](https://img.shields.io/badge/license-LGPL-brightgreen.svg)](LICENSE)
+[![PyPI Package](https://img.shields.io/pypi/v/molecule-hetznercloud)](https://pypi.org/project/molecule-hetznercloud/)
+[![License](https://img.shields.io/badge/license-LGPL-brightgreen.svg)](LICENSE)
 
-A [Hetzner Cloud](https://www.hetzner.com/cloud) plugin for [Molecule](https://molecule.readthedocs.io/en/latest/).
+A [Hetzner Cloud](https://www.hetzner.com/cloud) driver for [Molecule](https://ansible.readthedocs.io/projects/molecule/).
 
-This plugin allows you to do `molecule init role myrolename -d hetznercloud`
-and have Molecule provision on-demand Hetzner Cloud VPSes of your choice for
-your integration testing. New VPSes will be automagically created and
-provisioned on each `molecule test` run, SSH keys are generated and managed
-internally and all resources are cleaned up regardless of whether the role
-under test succeeds or fails.
+This plugin allows you to use on-demand Hetzner Cloud servers for your molecule integration tests.
 
 ## Install
 
 ```bash
 $ pip install molecule-hetznercloud
 ```
 
-If you're looking for a container approach, see [ansible-community/toolset](https://github.com/ansible-community/toolset).
-
 ## Upgrade
 
-Please see the [CHANGELOG.md](./CHANGELOG.md) for migration guides.
+Please see the [CHANGELOG.md](./CHANGELOG.md) for migration guides. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
I could not find any migration guides in the `CHANGELOG`. Should this be removed?

```suggestion
This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```